front_end/html/edit_course.html

```diff
@@ -56,6 +56,32 @@ <h2>Create course</h2>
     <p><textarea name="passcode" class="textarea is-grey is-medium is-fullwidth monospace" rows="1">{% if course_details["passcode"] %}{{ course_details["passcode"] }}{% end %}</textarea></p>
 </div>
 
+<div class="shadow-box">
+    <div>
+        <p><strong>Enable research: </strong></p>
+            <div class="select is-grey">
+                <select name="enable_research" id="enable_research" class="edit-select" onchange="showConsentForm()">
+                {% if course_details["consent_text"] and course_details["consent_text"] != "" %}
+                    <option value="Yes" selected>Yes</option>
+                    <option value="No">No</option>
+                {% else %}
+                    <option value="Yes">Yes</option>
+                    <option value="No" selected>No</option>
+                {% end %}
+                </select>
+            </div>
+
+            <div id="consent_form_div" class="top-space" {% if course_details['consent_text'] is None or course_details['consent_text'] == '' %} style="display:none" {% end %}>
+                <p><strong>Research consent form*: </strong><br />Please use <a href="https://www.markdownguide.org/basic-syntax/" target='_blank' rel='noopener noreferrer'>Markdown syntax</a> to write a consent form for this exercise.</p>
```

Comment:
Please add: "Students will review this form and indicate whether they consent to participate in the research."